### ISSUE TITLE
Add Indexes for Garage Zones and Save Garages to SA Garage Names

### ISSUE THREAD

**Issue by OrionSR** [sannybuilder/docs#3]
This data is a bit out of place for SCM reference but is very useful for managing garage memory with cleo scripts. Documentation for garages is seldom presented in index order so this information is hard to find anywhere else.

I figured out how to download the help-system repository using Visual Studio Code and was experimenting with adding the data but got distracted trying to hunt down a conversion strategy that could maintain the integrity of the HHP format. No luck so far.

Anyway, I think I can figure out how to squeeze in the info if I shorted the garage descriptions a little, so I'm wondering what's the best way to submit something.

**Comment by x87** [comment]
> BTW, reference for the HTML Help Workshop specified that the latest help technology was 1.4, and latest Workshop was 1.3. Your reference suggested v4.0 which I don't think ever existed.

I was referring to this:

![html-hw4](https://user-images.githubusercontent.com/5698288/86992662-f853e600-c16f-11ea-816b-599db4eb6407.png)

**Comment by OrionSR** [comment]
Yeah, the version info differs depending on where you look. I download the latest version (1.3, iirc) and got something similar to your report; last few entries are for 10.something though.

https://docs.microsoft.com/en-us/previous-versions/windows/desktop/htmlhelp/microsoft-html-help-downloads
Note

The latest version of the HTML Help technology is 1.4, as is the SDK. The latest version of HTML Help Workshop is 1.3. As of June 2009 there is no HTML Help Workshop 1.4.
